code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -456,6 +456,8 @@ Difficulty: Hard
 				else
 					burst_range = 3
 					INVOKE_ASYNC(src, .proc/burst, get_turf(src), 0.25) //melee attacks on living mobs cause it to release a fast burst if on cooldown
+				if(L.stat == CONSCIOUS && L.health >= 30)
```

Comment:
if you can outheal it you can probably tone down that healing so you stay at low hp and are still otherwise able to 'tank' it, or at least dance between 29-0.

It should just in general apply the burst on top of open fire if you ever get into melee and aren't in crit. Don't get into melee!

Reply:
Kk